### ISSUE TITLE
Feature request: emojis

### ISSUE THREAD

**Issue by TechWiz-3** [homeport/termshot#71]
Love this project. Would be great if you could add support for basic unicode emojis 🙏

**Comment by HeavyWombat** [comment]
> Love this project. Would be great if you could add support for basic unicode emojis 🙏

Good point. Absolutely. I think I have to rethink some internal decisions, because I am not quite sure how feasible it is. Same applies to support for Chinese characters.

**Comment by TechWiz-3** [comment]
Cool thanks

**Comment by jcraftsman** [comment]
I look forward to when this feature is ready. 😊
Nice tool by the way. Thx!
